samples/Basic_WiFi/app/application.cpp

```diff
@@ -41,6 +41,18 @@ void connectFail(const String& ssid, MacAddress bssid, WifiDisconnectReason reas
 	Serial.print(ssid);
 	Serial.print(_F("\", reason: "));
 	Serial.println(WifiEvents.getDisconnectReasonDesc(reason));
+
+	/*
+	 * Print available access points
+	 * 
+	 * Note: Calling this in init() *may* work, but it also may result in an error,
+	 * "STA is connecting, scan are not allowed!" (ESP32).
+	 *
+	 * The station interface must be enabled, however, so this is a good place to make the call.
+	 * 
+	 * A real application would normally do this elsewhere, for example after a user as logged in via AP.
```

Comment:
> after a user as logged

Did you mean `after a user has logged` ?

Reply:
logged in = authenticated

Reply:
No, I meant `after a user HAS logged in` instead of `after a user AS logged in`?

Reply:
Apologies, I was writing in Cockney. Will update :-)